pkg/apis/serving/v1alpha1/service_lifecycle_test.go

```diff
@@ -703,7 +703,7 @@ func TestAnnotateUserInfo(t *testing.T) {
 	for _, test := range tests {
 		test := test
 		t.Run(test.name, func(t *testing.T) {
-			t.Parallel()
+			// 			t.Parallel()
```

Comment:
Revert?

Reply:
Yes, the last two commits will be reverted.
They are needed to have -mesh E2E tests pass.